BitwardenShared/UI/Platform/Application/Views/NumberedList.swift

```diff
@@ -0,0 +1,79 @@
+import SwiftUI
+
+// MARK: - NumberedList
+
+/// A view that displays a numbered list of views, separated by a divider. The list has the
+/// secondary background color applied with rounded corners.
+///
+/// Adapted from: https://movingparts.io/variadic-views-in-swiftui
+///
+struct NumberedList<Content: View>: View {
+    // MARK: Properties
+
+    /// The content to display in the numbered list. Each child view will receive it's own number.
+    let content: Content
+
+    // MARK: View
+
+    var body: some View {
+        _VariadicView.Tree(Layout()) {
```

Comment:
is this part of swiftui's private view api?  I should make a ticket to use this within `PasswordAutoFillView`

Reply:
Yeah, it is a private SwiftUI API, but it's used pretty extensively within SwiftUI. There's a few places in the app that we could make use of this after it's in.